### DISCUSSION TITLE
Get the length of element matches from the get command

### DISCUSSION THREAD

**Discussion by chrisenitan** [cypress-io/cypress#14171] [Questions and Help]
Using the Cypress [get command](https://docs.cypress.io/api/commands/get.html#Syntax) in the test runner, cypress outputs the amount of element matches found for the selector asked for. Is there a way to get this value or length? 

- Tried to but could not via the console output array
![Screen Shot 2020-12-14 at 12 16 50](https://user-images.githubusercontent.com/11727537/102075246-45667300-3e06-11eb-94f0-657d665e5d83.png)

**Comment by jennifer-shehane** [comment]
You can use the `length` property

```js
cy.get('button').its('length').should('eq', 4)
cy.get('button').its('length').then((length) => {
  expect(length).to.eq(4)
})
cy.get('button').then(($buttons) => {
  expect($buttons.length).to.eq(4)
})
```

<img width="435" alt="Screen Shot 2020-12-14 at 6 07 20 PM" src="https://user-images.githubusercontent.com/1271364/102077142-3ba93380-3e37-11eb-954c-2676069cc2f7.png">

**Reply by chrisenitan** [reply]
Thanks, but I was looking more at a debugging approach so not really an assertion but more of getting the actual match value returned (if it is). So what I would be looking for is something like:

`cy.get("#button").match("length")`

..then I can use this however
